README.md

```diff
@@ -5,103 +5,111 @@
 
 [Augury](https://augury.angular.io/) is a Google Chrome Dev Tools extension for debugging Angular 2 applications. You can install the extension from [Chrome Store](https://chrome.google.com/webstore/detail/augury/elgalmkoelokbchhkhacckoklkejnhcd).
 
-Once the extenion is installed you can test it against the demo application https://augury.angular.io/demo/
 
-![Screenshot of Augury](assets/screenloop.gif)
+## Supported version
 
-## Supported Version
+Augury works with application built starting with Angular 2.0.
 
-Currently works with applications built in [Angular 2.0.0](https://github.com/angular/angular/blob/master/CHANGELOG.md#200-2016-09-14) using the Angular Component Router version `3.0.0-beta.2`.
+# Known issues
+
+## Router graph
+
+To be able to view the router graph, you will need to inject the Router in the application _Root_ component as shown below (it must be named `router` exactly).
 
-To view the router graph inject the Router in the application root component as shown below (it must be named `router` exactly).
```

Comment:
About the using `chore` for branch names per Rangle Git guidlelines.
![200](https://cloud.githubusercontent.com/assets/439226/22789596/6a57dde6-eeb1-11e6-8e8e-46f6fe56331e.gif)